src/components/CartItem.test.js

```diff
@@ -4,31 +4,32 @@ import CartItem from "./CartItem";
 
 describe("<CartItem/>", () => {
   const setup = () => {
-  const removeFromCartMock = jest.fn();
-  const item = { name: "item1", description: "something here" };
-  render(<CartItem item={item} removeFromCart={removeFromCartMock}></CartItem>);
-  return { removeFromCartMock, item}
-  }
-  
+    const removeFromCartMock = jest.fn();
+    const item = { name: "item1", description: "something here" };
+    render(
+      <CartItem item={item} removeFromCart={removeFromCartMock}></CartItem>
```

Comment:
Uma dica aqui, quando o componente não tem outros componentes filho, você pode usar uma tag auto-fechada:

```suggestion
      <CartItem item={item} removeFromCart={removeFromCartMock} />
```